tests/components/lamarzocco/snapshots/test_diagnostics.ambr

```diff
@@ -2,297 +2,23 @@
-      'enable_preinfusion': False,
-      'fri_auto': 'Disabled',
-      'fri_off_time': '00:00',
-      'fri_on_time': '00:00',
-      'global_auto': 'Enabled',
-      'mon_auto': 'Disabled',
-      'mon_off_time': '00:00',
-      'mon_on_time': '00:00',
-      'power': True,
-      'prebrewing_toff_k1': 5,
-      'prebrewing_toff_k2': 5,
-      'prebrewing_toff_k3': 5,
-      'prebrewing_toff_k4': 5,
-      'prebrewing_ton_k1': 3,
-      'prebrewing_ton_k2': 3,
-      'prebrewing_ton_k3': 3,
-      'prebrewing_ton_k4': 3,
-      'preinfusion_k1': 4,
-      'preinfusion_k2': 4,
-      'preinfusion_k3': 4,
-      'preinfusion_k4': 4,
-      'sat_auto': 'Disabled',
-      'sat_off_time': '00:00',
-      'sat_on_time': '00:00',
-      'steam_boiler_enable': True,
-      'steam_boiler_on': True,
-      'steam_level_set': 3,
-      'steam_set_temp': 128,
-      'steam_temp': 113,
-      'sun_auto': 'Disabled',
-      'sun_off_time': '00:00',
-      'sun_on_time': '00:00',
-      'thu_auto': 'Disabled',
-      'thu_off_time': '00:00',
-      'thu_on_time': '00:00',
-      'total_flushing': 69,
-      'tue_auto': 'Disabled',
-      'tue_off_time': '00:00',
-      'tue_on_time': '00:00',
-      'water_reservoir_contact': True,
-      'wed_auto': 'Disabled',
-      'wed_off_time': '00:00',
-      'wed_on_time': '00:00',
+      '__type': "<class 'lmcloud.models.LaMarzoccoMachineConfig'>",
+      'repr': "LaMarzoccoMachineConfig(turned_on=True, doses={<PhysicalKey.A: 1>: 135, <PhysicalKey.B: 2>: 97, <PhysicalKey.C: 3>: 108, <PhysicalKey.D: 4>: 121}, boilers={<BoilerType.STEAM: 'SteamBoiler'>: LaMarzoccoBoiler(enabled=True, current_temperature=123.80000305175781, target_temperature=123.9000015258789), <BoilerType.COFFEE: 'CoffeeBoiler1'>: LaMarzoccoBoiler(enabled=True, current_temperature=96.5, target_temperature=95)}, prebrew_mode=<PrebrewMode.PREINFUSION: 'TypeB'>, plumbed_in=True, prebrew_configuration={<PhysicalKey.A: 1>: LaMarzoccoPrebrewConfiguration(on_time=0.5, off_time=1), <PhysicalKey.B: 2>: LaMarzoccoPrebrewConfiguration(on_time=0.5, off_time=1), <PhysicalKey.C: 3>: LaMarzoccoPrebrewConfiguration(on_time=3.299999952316284, off_time=3.299999952316284), <PhysicalKey.D: 4>: LaMarzoccoPrebrewConfiguration(on_time=2, off_time=2)}, dose_hot_water=8, water_contact=True, wake_up_sleep_entries={'Os2OswX': LaMarzoccoWakeUpSleepEntry(enabled=True, days=[<WeekDay.MONDAY: 'monday'>, <WeekDay.TUESDAY: 'tuesday'>, <WeekDay.WEDNESDAY: 'wednesday'>, <WeekDay.THURSDAY: 'thursday'>, <WeekDay.FRIDAY: 'friday'>, <WeekDay.SATURDAY: 'saturday'>, <WeekDay.SUNDAY: 'sunday'>], entry_id='Os2OswX', steam=True, time_on='22:0', time_off='24:0'), 'aXFz5bJ': LaMarzoccoWakeUpSleepEntry(enabled=True, days=[<WeekDay.SUNDAY: 'sunday'>], entry_id='aXFz5bJ', steam=True, time_on='7:0', time_off='7:30')}, smart_standby=LaMarzoccoSmartStandby(enabled=True, minutes=10, mode=<SmartStandbyMode.LAST_BREWING: 'LastBrewing'>), brew_active=False, brew_active_duration=0)",
```

Comment:
Is this a dataclass? Should we use `asdict` before returning it? This is human readable, but it could be improved.